usersync/syncersbuilder.go

```diff
@@ -80,6 +80,20 @@ func BuildSyncers(hostConfig *config.Configuration, bidderInfos config.BidderInf
 	return syncers, nil
 }
 
+func shouldCreateSyncer(cfg config.BidderInfo) bool {
+	if !cfg.Enabled {
+		return false
+	}
+
+	if cfg.Syncer == nil {
+		return false
+	}
+
+	// a syncer may provide just a Supports field to provide hints to the host. we should only try to create a syncer
+	// if there is at least one non-Supports value populated.
+	return cfg.Syncer.Key != "" || cfg.Syncer.Default != "" || cfg.Syncer.IFrame != nil || cfg.Syncer.Redirect != nil || cfg.Syncer.SupportCORS != nil
```

Comment:
This is a bit hacky. Perhaps it would be better to use different data structures for yaml vs config overrides... but I think this works.